### ISSUE TITLE
Delete has no effect

### ISSUE THREAD

**Issue by Misco** [terminal42/contao-fineuploader#96]
Using Contao 4.13 in a own DCA with this config:

`$GLOBALS['TL_DCA']['tl_member']['fields']['video_vorschau'] = [
    'label'     => ['Video Vorschaubild', ''],
    'exclude'   => true,
    'inputType' => 'fineUploader',
    'eval'      => [
        'multiple'          => false,
        'storeFile'         => true,
        'uploadFolder'      => 'files/Videos auf Detailseite/Vorschaubilder',
        'useHomeDir'        => false,
        'uploaderLimit'     => 1,
        'maxConnections'    => 1,
        'addToDbafs'        => true,
        'doNotOverwrite'    => true,
        'debug'             => true,

        // Validation
        'extensions'        => 'jpg,jpeg,png,webm',
        'minlength'         => 0,
        'maxlength'         => 2048000,
        
        // Chunking
        'chunking'          => false,

        // Rendering
        'isGallery'         => true,
        'isDownloads'       => false,
        'sortable'          => false,
        'uploadButtonTitle' => '',

        'directUpload'      => true,
    ],
    'sql' => "blob NULL"
];`

When i click the red cross, the entry goes away, but the file remains.
What can i do? Thanks

**Comment by tn34de** [comment]
Hallo zusammen,
das Problem haben wir auch bemerkt.
Ich versuche gerade eine Validierung der Größe aller Dateien per validateFormField-Hook umzusetzen. Das wäre soweit auch in Ordnung und würde funktionieren, wenn der fine-uploader die Dateien auch in der $_SESSION reduzieren würde, wenn man sie per X entfernt. Selbst nachdem die Liste leer ist zeigt ein DEBUG-Output, dass immer noch alle Dateien in $_SESSION['FILES'] vorhanden sind. Dafür gibt es leider keinen Workaround. 

Wieviel würde es kosten, wenn wir die Beseitigung dieses Bugs in Auftrag geben würden?
Danke und Gruß

Hello all,
we have noticed the problem as well.
I am trying to implement a validation of the size of all files via validateFormField hook. This would be fine so far and would work if the fine-uploader would also reduce the files in the $_SESSION when removing them via X. Even after the list is empty, a DEBUG output shows that all files are still present in $_SESSION['FILES']. Unfortunately there is no workaround for this. 

How much would it cost to commission the removal of this bug?
Thanks and greetings


Andy / TN34.DE GmbH & Co. KG

**Comment by aschempp** [comment]
`fineuploader` is an uploader, not a file manager. I don't think it is ever inteded to delete anything from the widget. That would neeed to be a custom implementation in your DCA.

**Comment by fritzmg** [comment]
That's not true, we use it this way in a few projects - but we haven't had any issues.
